test/interpreter_functional/plugins/kbn_tp_run_pipeline/public/components/main.js

```diff
@@ -55,6 +55,7 @@ class Main extends React.Component {
 
     window.renderPipelineResponse = async (context = {}) => {
       return new Promise(resolve => {
+        props.visualizationLoader.destroy(this.chartDiv);
```

Comment:
Makes sure previous visualisation is cleaned up before expected error to capture "empty" screenshot